canvas_modules/common-canvas/__tests__/common-canvas/common-canvas-test.js

```diff
@@ -130,9 +164,10 @@ describe("CommonCanvas renders correctly", () => {
 		const toolbarConfig = [{ action: "palette", label: "Palette", enable: true }];
 		const notificationConfig = { action: "notification", label: "Notifications", enable: true };
 		const config = {};
+		const canvasParams = {};
```

Comment:
There's no need to do this now that the function can handle undefined canvasParmas.